frontend/src/lib/utils.ts

```diff
@@ -1089,3 +1089,13 @@ export function validateFileExtension(ext: string) {
 	const validExtensionRegex = /^[a-zA-Z0-9]+([._][a-zA-Z0-9]+)*$/
 	return validExtensionRegex.test(ext)
 }
+
+export function isFlowPreview(job_kind: Job['job_kind'] | undefined) {
```

Comment:
What do you think of `is{Flow,Script}PreviewLike` ?
And maybe a small doc comment like "used where job kind was checked for preview but new kinds were added in the backend with same meaning for the frontend"